test/jdk/ProblemList-fips.txt

```diff
@@ -254,7 +256,7 @@ sun/security/krb5/ktab/BufferBoundary.java https://github.com/ibmruntimes/openj9
 sun/security/krb5/ktab/FileKeyTab.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
 sun/security/krb5/ktab/KeyTabIndex.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
 sun/security/krb5/RFC396xTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
-sun/security/krb5/runNameEquals.sh.runNameEquals
+sun/security/krb5/runNameEquals.sh https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-x64
```

Comment:
I think all instances of `linux-x64` should be replaced by `generic-all`.
Shouldn't the tests should be excluded for any platform that can operate in FIPS mode?

Reply:
I think only linux-64 platform support FIPS mode for now, it can be replaced by `generic-all` in the future.